### ISSUE TITLE
ECONFLICT Unable to find suitable version for loader.js

### ISSUE THREAD

**Issue by tsteuwer** [amkirwan/ember-oauth2#19]
Not entirely sure if this is you or loader.js, but:

```
bower install
bower ember-oauth2#*            cached https://github.com/amkirwan/ember-oauth2.git#0.5.5
bower ember-oauth2#*          validate 0.5.5 against https://github.com/amkirwan/ember-oauth2.git#*
bower loader.js#1.0.1           cached git://github.com/stefanpenner/loader.js.git#1.0.1
bower loader.js#1.0.1         validate 1.0.1 against git://github.com/stefanpenner/loader.js.git#1.0.1
bower                        ECONFLICT Unable to find suitable version for loader.js
```

Also, EmberCLI uses ember-cli/loader.js#3.2.0 would this end up causing issues?

**Comment by amkirwan** [comment]
loader.js is now in ember-cli/loader.js. I am going to release a new version today or tomorrow with some updates.

**Comment by amkirwan** [comment]
fixed in the v0.6.0 release
